### DISCUSSION TITLE
Request Bodies

### DISCUSSION THREAD

**Discussion by kinlane** [postman-open-technologies/linting-rules#17] [General]
This is a discussion around the request boies ruleset, keeping an ongoing record of what has gone into the ruleset.

**Comment by jwbanning** [comment]
The linting rule "DELETE methods must have a request body." should read "DELETE methods must NOT have a request body."
The message and description in the image below give mixed messages. 

<img width="861" alt="Screen Shot 2021-11-30 at 5 29 53 PM" src="https://user-images.githubusercontent.com/2584258/144144554-db0682f7-b17c-443b-ad11-deb301405e79.png">

Looking at: https://rules.linting.org//rules/request-bodies-delete-oas3/

**Comment by jwbanning** [comment]
The linting rule "GET methods must have a request body." should read "GET methods must NOT have a request body."
The message and description in the image below give mixed messages.

<img width="612" alt="Screen Shot 2021-11-30 at 5 35 26 PM" src="https://user-images.githubusercontent.com/2584258/144145027-152234e6-602b-420e-8b48-1558b2163552.png">
